connection.go

```diff
@@ -116,9 +116,9 @@ func (d defaultLogger) Report(event ConnLogKind, conn *Connection, v ...interfac
 //
 // ATTENTION: result argument for *Typed methods should deserialize from
 // msgpack array, cause Tarantool always returns result as an array.
-// For all space related methods and Call* (but not Call17*) methods Tarantool
+// For all space related methods and Call16* (but not Call*) methods Tarantool
```

Comment:
`Call` still works as `Call16` by default, so the replacement of `Call17` to `Call` looks premature. I would use explicit `Call16` and `Call17` in the documentation.

Reply:
Fixed.